eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/model/DbsnpJson.java

```diff
@@ -0,0 +1,74 @@
+package uk.ac.ebi.eva.accession.dbsnp.model;
+
+import org.springframework.batch.core.configuration.annotation.StepScope;
+
+/**
+ * Dbsnp object model to store key fields from Dbsnp Json
```

Comment:
```suggestion
 * Core fields of a variant extracted from a dbSNP JSON file.
```

Reply:
Corrections noted (and those from above as well)!